src/components/types.ts

```diff
@@ -26,6 +26,14 @@ export interface ItemImage {
   alt: string;
 }
 
+export interface GetCustomNotes {
```

Comment:
@itbel as I mentioned in my other comment, you can probably remove this, otherwise good work 👌

Reply:
I must have forgotten to remove it when I was playing with implementation. Thanks for the help!